install/installer/traf_config_setup

```diff
@@ -226,6 +230,33 @@ echo "export NODE_LIST=\"$NODE_LIST\"" >> $LOCAL_TRAF_CONFIG
 node_count=$(echo $NODE_LIST | wc -w)
 
 #==============================================
+echo -n "Are management nodes enabled (Y/N), default is N: "
```

Comment:
Should this question be "Do you have a set of management nodes (Y/N), default is N:"